pkg/client/unversioned/portforward/portforward_test.go

```diff
@@ -206,7 +206,7 @@ func TestGetListener(t *testing.T) {
 }
 
 // fakePortForwarder simulates port forwarding for testing. It implements
-// kubeletserver.PortForwarder.
+// portforward.PortForwarder.
```

Comment:
Ack. This is just a temporary commit copied from https://github.com/kubernetes/kubernetes/pull/35329, which is a straight move. In the interest of keeping it a minimal move, I'd prefer not to touch any parts I don't need to in that PR.